deltachat-ios/View/ContactCell.swift

```diff
@@ -136,6 +148,25 @@ class ContactCell: UITableViewCell {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
         setupSubviews()
         selectionStyle = .default
+        configureCompressionPriority()
+    }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        if previousTraitCollection?.preferredContentSizeCategory !=
+            traitCollection.preferredContentSizeCategory {
+            configureCompressionPriority()
+        }
+    }
+
+
+    private func configureCompressionPriority() {
+        if isLargeText {
```

Comment:
For very large font sizes (larger than normal scale in system font size settings, avaliable only via accessibility settings),  the timelabel can be overlapped by the chat title. for small to large font sizes the time label always overlapps the chat title if there's not enough place for both. This was also the default before